chatbot/setup_postgresql_local.sh

```diff
@@ -0,0 +1,9 @@
+sudo -u postgres psql -p $PG_PORT -c "CREATE ROLE $PG_USER PASSWORD '$PG_PASSWORD' LOGIN"
+sudo -u postgres createdb -p $PG_PORT -O $PG_USER $PG_USER
+sudo -u postgres psql -p $PG_PORT -d $PG_USER -c 'CREATE EXTENSION IF NOT EXISTS "pgcrypto"'
```

Comment:
To keep the Server and Local versions of this file in sync, could we have `setup_postgresql_local.sh` call `setup_postgresql_server.sh` for the rest? Then it will be clear that the only difference between the two is the user and role setup, and when we add a new migration script we'll only need to add it in one place.

In which case, what do you think about changing the name of `setup_postgresql_server.sh` back to just `setup_postgresql.sh` since it will be used both locally and on the server? I could go either way on this thought.

Reply:
aha! keeping things in sync is a problem I didn't think about. I like both the call from one script to the other and the more appropriate naming as a consequence

Reply:
@trisapeace this is now part of the pull request!